javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -213,6 +213,17 @@ object Character {
     else -1
   }
 
+  // ported from https://github.com/gwtproject/gwt/blob/master/user/super/com/google/gwt/emul/java/lang/Character.java
+  def forDigit(digit: Int, radix: Int): Char = {
+    if ( radix < MIN_RADIX || radix > MAX_RADIX || digit < 0 || digit >= radix) {
+      0
+    }
+    else {
+      val overBaseTen = digit - 10
+      (if (overBaseTen < 0) '0' + digit else 'a' + overBaseTen).toChar
```

Comment:
Have you considered using JavaScript's [`toString`][1]? As far as I can see, it does exactly what we want. To enable it, import `js.JSNumberOps.enableJSNumberOps`.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString